src/swsssdk/configdb.py

```diff
@@ -193,6 +193,42 @@ def set_entry(self, table, key, data):
                     k = k + '@'
                 client.hdel(_hash, self.serialize_key(k))
 
+    def set_bulk(self, payload):
+        """Write bulk entries to config db.
+        """
+        client = self.redis_clients[self.db_name]
```

Comment:
You need to use the below API which is Multi-DB compliant.

self.get_redis_client(self.db_name). Modify other occurrences as well.